app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -159,4 +175,44 @@ class AddDeadlineActivity : AppCompatActivity() {
             }
         }
     }
+
+    private fun selectPDF() {
+        PDFUtil.selectPdfIntent() {
+            startForResult.launch(it)
+        }
+    }
+
+    //handle result of startActivity
+    val startForResult =
+        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result: ActivityResult ->
+            if (result.resultCode == Activity.RESULT_OK) {
+                val intent = result?.data!!
+                pdfUri = intent.data!!
+                val uriString: String = pdfUri.toString()
+                var pdfName: String? = null
```

Comment:
Why this declaration here? `pdfName` could simply be a `val` declared when you initialize it I think.